### DISCUSSION TITLE
Assinatura para o contrato da Python Norte 2024

### DISCUSSION THREAD

**Discussion by meu-bem** [apyb/comunidade#176] [Geral]
Olá, @anicelysantos. Tudo bem?
O Eldorado já mandou o contrato do patrocínio para assinatura. Já validamos e ficamos no aguardo da tua resposta.
Mais uma vez, bringadão, mesmo!

**Comment by anicelysantos** [comment]
Oi @meu-bem obrigada por avisar. Assinado! :)
